core/templates/pages/error-pages/error-iframed-page/error-iframed-page.import.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Scripts for the error iframed page.
+ */
+import 'core-js/es7/reflect';
+import 'zone.js';
+
+// TODO(#13080): Remove the mock-ajs.ts file after the migration is complete.
+import 'pages/mock-ajs';
+import 'Polyfills.ts';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { AppConstants } from 'app.constants';
+import { enableProdMode } from '@angular/core';
+import { ErrorIframedPageModule } from './error-iframed-page.module';
+
+if (!AppConstants.DEV_MODE) {
+  enableProdMode();
+}
+
+platformBrowserDynamic().bootstrapModule(ErrorIframedPageModule).catch(
+  // eslint-disable-next-line no-console
+  (err) => console.error(err)
```

Comment:
Are we going to have thisflag in all the import files?

Reply:
yeah, but there will be only 4 import files after completion of migration to the angular and angular router.

- oppia-root.import.ts
- error-page.import.ts
- error-iframed-page.import.ts
- maintenance-page.import.ts

Reply:
Can we use the loggerService [Trying to remove the bypass flag]

Reply:
We can't use logger service before bootstrapping of angular application.